fenicsxconcrete/experimental_setup/concrete_cylinder.py

```diff
@@ -0,0 +1,248 @@
+from fenicsxconcrete.experimental_setup.base_experiment import Experiment
+from fenicsxconcrete.helper import Parameters
+import dolfinx as df
+import numpy as np
+import gmsh
+import os
+import meshio
+import ufl
+from mpi4py import MPI
+
+from fenicsxconcrete.unit_registry import ureg
+from fenicsxconcrete.boundary_conditions.boundary import plane_at, point_at
+from fenicsxconcrete.boundary_conditions.bcs import BoundaryConditions
+
+
+def generate_cylinder_mesh(radius, height, mesh_density):
```

Comment:
Please have a look at the implementation in fenics_concrete. The problem there with the cylinder is that due to the approximation error of the mesh, the area of the crossection did not correspond to the area of the circle.
is this fixed in fenicsx (quadratic geometry?), otherwise the same feature would be nice.

Reply:
This can be fixed in dolfinx, when you build the gmsh mesh with quadratic elements. Do we just want to make this mandatory for round geometries?

Reply:
Isn't that what I implemented in lines 131-159? The update is done comparing the area as in the original implementation and modifying the radius, but we can enforce the quadratic elements as well.

Reply:
I guess the most flexible way would be to leave it and depending on the element type the radius is changed or not.
But the cleanest, probably most practical way would be just enforce quadratic elements, or even just live with the error if someone really want to use linear elements.

Reply:
I added the option to indicate tthe degree of the mesh in the generation of the cylinder and set the default to 2. If the degree is 1, it will run the update, otherwise not.

Reply:
Sounds great